tests/browser/test/test_setup.js

```diff
@@ -126,6 +142,13 @@ async function getCategory(browser, categoryName) {
   return category;
 }
 
+/**
+ * @param browser The active WebdriverIO Browser object.
+ * @param categoryName The name of the toolbox category to search.
+ * @param n Which block to select, indexed from the top of the category.
```

Comment:
```suggestion
 * @param n Which block to select, 0-indexed from the top of the category.
```

Not sure if this is necessary since it's convention, but migth be worth clarifying!